python/nano/test/run-nano-test.sh

```diff
@@ -20,6 +20,11 @@ python -m pytest -s ${NANO_TEST_DIR}/test_models_vision.py
 now=$(date "+%s")
 time2=$((now-start))
 
+echo "#2 Start test lightning"
```

Comment:
let's change this scripts to test one directory.

Reply:
I will open another issue and PR to resolve this.

Reply:
Will resolve this here: https://github.com/intel-analytics/BigDL/issues/3762